src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -1104,8 +1104,8 @@ def write_to_app_pipe(self, command_dict: dict, app_pipe_prefix: Optional[str] =
                 logger.info(f"Sending out-of-band command: {command} to file: {app_pipe_name}")
                 app_pipe.write(json.dumps(command_dict) + "\n")
 
-            # TODO(#31239): remove the need for sleep
-            sleep(0.001)
+            # TODO(#31239): remove the need for sleep, till then let's wait 20ms
```

Comment:
Maybe consider not saying the actual amount (so that if we update the value we do not need to keep comment watched to the value).

Beyond that, could you add in a comment what was considered how to pick the value? Is it "this value was currently tested on matter.js" or similar? This comment does not seem to explain what a "good value" is

Reply:
@andy31415 done. And yes in fact I did several runs and 20ms is reliable enough to run tests stable.